### PR TITLE
Feat: added result page of the quiz app

### PR DESCRIPTION
**Pull request by vivekmehto**
image

### CODE REVIEW

**Review by aishwarya-bm — APPROVED** [review]
Looks all good and clean, the naming & the styles. Good to merge

File: css/result.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.result-container {
+  font-size: 1.5rem;
+}
+
+.result-container .list li:hover {
+  color: var(--black);
+  cursor: default;
```
Comment (ttahm3d) [inline css/result.css:7]: are you overriding here?

**Review by ttahm3d — APPROVED** [review]
looks good. Can be merged

**Review by Kothalkarpriya — APPROVED** [review]
Good work! You can merge

File: css/result.css (hunk 1 above)
Comment (vivekmehto) [reply css/result.css:7]: yes

**Pull request merged**